charms/istio-gateway/tests/conftest.py

```diff
@@ -0,0 +1,75 @@
+import pytest
```

Comment:
this looks like the beginning of a unit test framework?  if so, could you push all the unit tests into ```.../tests/unit/conftest.py``` so there's room for integration tests (whenever they should come?). If there is already integration tests elsewhere -- feel free to ignore

Reply:
The tests for both charms have been refactored into their own unit/ directory within tests/

Reply:
The integration tests live at the top (bundle) level because they depend on (and cover) both charms.